src/demangle.h

```diff
@@ -77,7 +77,7 @@ _START_GOOGLE_NAMESPACE_
 // Demangle "mangled".  On success, return true and write the
 // demangled symbol name to "out".  Otherwise, return false.
 // "out" is modified even if demangling is unsuccessful.
-bool GOOGLE_GLOG_DLL_DECL Demangle(const char *mangled, char *out, int out_size);
```

Comment:
The error I was getting was "./src/demangle.h:80:26: error: expected ';' after top level declarator"

Reverting the change I can't reproduce that error now, so one of the other changes may have somehow fixed this...